docs/content/routing/providers/kubernetes-ingress.md

```diff
@@ -8,6 +8,12 @@ description: "Understand the routing configuration for the Kubernetes Ingress Co
 The Kubernetes Ingress Controller.
 {: .subtitle }
 
+!!! info ""
+
+    It is not possible to route requests directly to [Kubernetes services](https://kubernetes.io/docs/concepts/services-networking/service/ "Link to Kubernetes service docs").
+
+    Please check the [communication between Traefik and Pods](https://community.traefik.io/t/communication-between-traefik-and-pods/17005 "Link to thread in the forum") thread in the community forum for more information.
```

Comment:
@svx I think we don't want to redirect to the community forum to have more information on that topic. We already have [this part](https://doc.traefik.io/traefik/routing/providers/kubernetes-ingress/#communication-between-traefik-and-pods) of the Ingress provider documentation that could fit. If the intention here was to have more insights than what's already there, why not also improve this part of the documentation we can refer to? WDYT?

Reply:
@rtribotte Thanks! I will have a look and will update my PR